card.go

```diff
@@ -706,7 +706,10 @@ func searchScryfallCard(cardTokens []string) ([]Card, error) {
 }
 
 func ParseAndFormatSearchResults(csr CardSearchResult) ([]Card, error) {
-	if len(csr.Data) <= 5 {
+	fullyShownThreshold := 2 // limit for how many results are fully displayed (inclusive)
```

Comment:
Can we move these to consts at the top of the file rather than variables inside the function?

Reply:
Sure, moved. I found it more readable to put them there as this is the only function where they're relevant, but if you prefer keeping all constants in one place, I don't care.